cypress/e2e/0-smoke.cy.ts

```diff
@@ -9,6 +9,7 @@ const lastName = randLastName();
 describe('Authentication', () => {
 	beforeEach(() => {
 		cy.task('db:reset');
+		Cypress.session.clearAllSavedSessions();
```

Comment:
We should instead change [`db:reset`](https://github.com/n8n-io/n8n/blob/master/cypress.config.js#L16) to 
```
				'db:reset': () =>
					Promise.all([
						fetch(BASE_URL + '/e2e/db/reset', { method: 'POST' }),
						Cypress.session.clearAllSavedSessions(),
					]),
```

Reply:
Unfortunately cannot use `Cypress.session` outside of test suites. I've renamed the tasks though.

Reply:
let's merge this as it for now.